TaskLayer/CalibrationTask/CalibrationTask.cs

```diff
@@ -332,7 +337,7 @@ private DataPointAquisitionResults GetDataAcquisitionResults(MsDataFile myMsData
 
             new FdrAnalysisEngine(allPsms, searchMode.NumNotches, CommonParameters, new List<string> { taskId, "Individual Spectra Files", fileNameWithoutExtension }).Run();
 
-            List<PeptideSpectralMatch> goodIdentifications = allPsms.Where(b => b.FdrInfo.QValueNotch < 0.01 && !b.IsDecoy && b.FullSequence != null).ToList();
+            List<PeptideSpectralMatch> goodIdentifications = allPsms.Where(b => b.FdrInfo.QValueNotch < 0.001 && !b.IsDecoy && b.FullSequence != null).ToList();
```

Comment:
any particular reason for this?

Reply:
It produced better calibration, since there was a lot of noise introduced by the lower confidence PSMs. Average MS1 PPM error went from -0.059 to -0.031 and standard deviation went from 0.91 to 0.87.
It's not a huge difference, so we can scrap it if you prefer, but it seemed that using higher confident data points achieved better calibration. It doesn't add any time, either.

I would think the advantage to using 0.01 instead of 0.001 would be for files with fewer PSMs in them.